src/Psalm/Internal/Analyzer/ClassAnalyzer.php

```diff
@@ -651,16 +652,30 @@ public static function addContextProperties(
                     ) {
                         if (IssueBuffer::accepts(
                             new OverriddenPropertyAccess(
-                                'Property ' . $guide_class_storage->name . '::$' . $property_name
+                                'Property ' . $fq_class_name . '::$' . $property_name
                                     . ' has different access level than '
                                     . $storage->name . '::$' . $property_name,
                                 $property_storage->location
                             )
                         )) {
                             // fall through
                         }
+                    }
 
-                        continue;
+                    if ($property_storage->type && $guide_property_storage->type && $property_storage->location &&
+                        !$property_storage->type->equals($guide_property_storage->type)) {
+                        if (IssueBuffer::accepts(
+                            new NonInvariantChildProperty(
+                                'Property ' . $fq_class_name . '::$' . $property_name
+                                . ' has type ' . $property_storage->type->getId()
+                                . ", not invariant with " . $guide_class_name . '::$' . $property_name . ' of type ' .
+                                $guide_property_storage->type->getId(),
+                                $property_storage->location
+                            ),
+                            $statements_source->getSuppressedIssues()
```

Comment:
You need to add $property_storage->suppressed_issues to allow it to be suppressed

Reply:
Thanks. `\Psalm\Storage\PropertyStorage` didn't have a `suppressed_issues` property so I've added it, not sure if I went about populating it in the right way.